src/components/Card.tsx

```diff
@@ -0,0 +1,20 @@
+import OpenColor from "open-color";
+
+import "./Card.scss";
+
+export const Card: React.FC<{
+  color: keyof OpenColor;
+}> = ({ children, color }) => {
+  return (
+    <div
+      className="Card"
+      style={{
+        ["--card-color" as any]: OpenColor[color][7],
+        ["--card-color-darker" as any]: OpenColor[color][8],
+        ["--card-color-darkest" as any]: OpenColor[color][9],
```

Comment:
we already have classname above , can we use that to define these css vars instead of adding inline styles here ?

Reply:
Using css variables is by far the easiest and most customizable solution. With classnames we'd have to hardcode the colors for each card color className.

Reply:
I meant defining the CSS vars inside the class `Card` instead of inline styles

Reply:
Which would be the same thing as using classes, no?

Reply:
Not sure what you meant ..., I mean something like 👇🏻 
```css
.Card {
    --card-color: $oc-lime0;
    --card-color-darker: $oc-lime1;
}    
```

Reply:
We'd need to have a class for each supported color:

```
.Card.Card--lime {
  --card-color: #{$oc-lime-7};
  --card-color-darker: #{$oc-lime-8};
  --card-color-darkest: #{$oc-lime-9};
}
.Card.Card--orange {
  --card-color: #{$oc-orange-7};
  --card-color-darker: #{$oc-orange-8};
  --card-color-darkest: #{$oc-orange-9};
}
/*...*/
```

Reply:
my bad sorry, 🤦🏻 I missed out that color is coming via props 🤦🏻